### PR TITLE
[#1] Update repo configs from https://getscaffold.dev/

### PR DESCRIPTION
**Pull request by tannguyen04**
## Checklist before requesting a review

#1 
 - [x] I have formatted the subject to include ticket number
   as `[#123] Verb in past tense with dot at the end.`
 - [x] I have added a link to the issue tracker
 - [x] I have provided information in `Changed` section about WHY something was
   done if this was not a normal implementation
 - [x] I have performed a self-review of my code
 - [x] I have commented my code, particularly in hard-to-understand areas
 - [x] I have added tests that prove my fix is effective or that my feature works
 - [x] I have run new and existing relevant tests locally with my changes, and
   they passed
 - [x] I have provided screenshots, where applicable

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        uses: luizm/action-sh-checker@v0.8.0
+        env:
+          SHFMT_OPTS: -i 2 -ci -s -d
+      - name: Test
+        run: |
+          git clone https://github.com/bats-core/bats-core.git
+          cd bats-core
+          sudo ./install.sh /usr/local
+          cd ..
+          bats --tap tests/bats/rename.bats
```
Comment (tannguyen04) [inline .github/workflows/test.yml:27]: Hi @AlexSkrypnyk 
Can you please look into this when you have time ?

The `bats` command it stuck in Github workflow, the circle spin non stop. But if I ssh to GitHub workflow then run `bats --tap tests/bats/rename.bats` it worked. 

And I also tested it on my local, the tests is passed.

Screenshot 2023-12-28 at 9 21 04 AM

Comment (AlexSkrypnyk) [reply .github/workflows/test.yml:27]: @tannguyen04 
Please use a different method to install bats

see example here
https://github.com/AlexSkrypnyk/scaffold/blob/7b6abfe186bd56d44dbc0a19721419f797245a8f/.github/workflows/scaffold-test.yml#L17

Comment (tannguyen04) [reply .github/workflows/test.yml]: Thanks @AlexSkrypnyk 

I tried install Bats from Github Action and from Source. Both of them are not working.
But it worked with NPM. I have updated the PR.
Could you please re-review ?

Comment (tannguyen04) [reply .github/workflows/test.yml]: https://github.com/tannguyen04/drupal-module-renamer/actions/runs/7353665817 Test passed.

**Review by AlexSkrypnyk — APPROVED** [review]

**Pull request merged**